### DISCUSSION TITLE
Regarding Copilot

### DISCUSSION THREAD

**Discussion by canocalir** [community/community#27951] [Copilot Conversations]
My copilot not working properly on vscode. Autocompletion is very problematic.

**Comment by yusuftemel** [comment]
This issue is caused by an extension, please file it with the repository (or contact) the extension has linked in its overview in VS Code or the [marketplace](https://aka.ms/vscodemarketplace) for VS Code. See also [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding! [canocalir](https://github.com/canocalir)
